src/runtime/contrib/dnnl/dnnl_json_runtime.cc

```diff
@@ -82,15 +82,511 @@ class DNNLJSONRuntime : public JSONRuntimeBase {
+                                                       {"chwn", tag::chwn},
+                                                       {"ncdhw", tag::ncdhw},
+                                                       {"ndhwc", tag::ndhwc},
+                                                       {"oi", tag::oi},
+                                                       {"io", tag::io},
+                                                       {"oiw", tag::oiw},
+                                                       {"owi", tag::owi},
+                                                       {"wio", tag::wio},
+                                                       {"iwo", tag::iwo},
+                                                       {"oihw", tag::oihw},
+                                                       {"hwio", tag::hwio},
+                                                       {"ohwi", tag::ohwi},
+                                                       {"ihwo", tag::ihwo},
+                                                       {"iohw", tag::iohw},
+                                                       {"oidhw", tag::oidhw},
+                                                       {"dhwio", tag::dhwio},
+                                                       {"odhwi", tag::odhwi},
+                                                       {"iodhw", tag::iodhw},
+                                                       {"idhwo", tag::idhwo},
+                                                       {"goiw", tag::goiw},
+                                                       {"gowi", tag::gowi},
+                                                       {"wigo", tag::wigo},
+                                                       {"gohwi", tag::gohwi},
+                                                       {"goihw", tag::goihw},
+                                                       {"hwigo", tag::hwigo},
+                                                       {"giohw", tag::giohw},
+                                                       {"goidhw", tag::goidhw},
+                                                       {"giodhw", tag::giodhw},
+                                                       {"godhwi", tag::godhwi},
+                                                       {"dhwigo", tag::dhwigo},
+                                                       {"tnc", tag::tnc},
+                                                       {"ntc", tag::ntc},
+                                                       {"ldnc", tag::ldnc},
+                                                       {"ldigo", tag::ldigo},
+                                                       {"ldgoi", tag::ldgoi},
+                                                       {"ldio", tag::ldio},
+                                                       {"ldoi", tag::ldoi},
+                                                       {"ldgo", tag::ldgo},
```

Comment:
tag_last?

Reply:
deleted. I intend to list all the tags in oneDNN header except for the `abcd...`.